docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -45,25 +45,23 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.services.entries.accessLabel** | No | Specifies the label used in Istio rules in Application Connector. This field is required for the API entry type. |
-| **spec.services.entries.targetUrl** |  No | Specifies the URL of a given API. This field is required for the API entry type. Parameter provided by Application Registry, do not edit. |
-| **spec.services.entries.oauthUrl** | No | Specifies the URL used to authorize with a given API. This field is required for the API entry type. Parameter provided by Application Registry, do not edit. |
-| **spec.services.entries.credentialsSecretName** | No | Specifies the name of the Secret which allows you to call a given API. This field is required if **spec.services.entries.oauthUrl** is specified. Parameter provided by Application Registry, do not edit. |
+| **spec.services.entries.targetUrl** |  No | Specifies the URL of a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
+| **spec.services.entries.oauthUrl** | No | Specifies the URL used to authorize with a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
+| **spec.services.entries.credentialsSecretName** | No | Specifies the name of the Secret which allows you to call a given API. This field is required if **spec.services.entries.oauthUrl** is specified. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |
 
 ## Related resources and components
 
 These components use this CR:
 
 | Component   |  Description |
 |-----------|-------------|
-| Application Registry | Reads and saves the APIs and Event Catalog metadata of the connected external solution in this CR. |
\ No newline at end of file
+| Application Gateway | Reads the API metadata in order to connect to the external system. | 
+| Runtime Agent (in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md)) | Saves the metadata of the connected external system in the Application CR, synchronises the metadata stored in Compass with the state in the cluster stored in the Application CR. |
```

Comment:
I think Central Application Connectivity validator should be also included here

Reply:
What does it do with the CR? If I include it, I also need to add a description of how CACV uses it/what it uses it for.

Reply:
I received input via PM:

1) 
> The Application Connectivity Validator validates requests from external system with the `X-Forwarded-Client-Cert` header that contains `Subject` with the following fields corresponding to the Application custom resource:
> - **CommonName** is the name of the Application custom resource.
> - (Optional) **Organization** is the tenant.

2)

> The Application Connectivity Validator validates requests/events from external system against respective Application custom resource

3)

> Application Connectivity Validator makes sure that the events originating from registered External Systems end up in the respective workload, blocking any events that are not sent with the proper mTLS certificate.

I decided to go with:
> | Application Connectivity Validator (in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md)) | Validates requests and events from the external system against the respective Application CR. |